profiling/src/lib.rs

```diff
@@ -475,13 +479,19 @@ extern "C" fn rinit(_type: c_int, _module_number: c_int) -> ZendResult {
         output_pprof,
     ) = unsafe {
         let profiling_enabled = config::profiling_enabled();
+        let profiling_experimental_features_enabled =
+            profiling_enabled && config::profiling_experimental_features_enabled();
         (
             profiling_enabled,
+            profiling_experimental_features_enabled,
```

Comment:
This section is getting a bit too messy for me, and it makes it hard to review. Could you spend a few minutes seeing if you can find a good way to clean this up? I'm happy to pair if you want to do it together.

Reply:
I moved this code upwards to the `config::profiling_*_enabled()` function, as this is where it belongs IMHO
Tell me what you think about it!